docs/6.-Engineering/Test-Engineering/Supported-Browsers,-Devices-&-Operating-Systems.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Supported Browsers, Devices & Operating Systems
+authors: 
+- Karen Farrell
+reviewed: [yyyy-mm-dd]
+reviewer:
+next-review:
+---
+
+
+## **Who is this for?**
+This page documents the browsers, devices and operating systems we support and therefore test against.  These must be considered when designing, coding, as well as testing AM applications.
+
+## **Introduction**
+With such a wide range of browsers, devices and operating systems available, cross browser testing has become a crucial part of the Software Development Lifecycle.
+
+It is important to compare a web applications functionality and design across multiple browsers and platforms to ensure consistent behaviour, functionality and user experience.
+
+Styling and effects such as mouse-over, pop-ups and fonts can vary significantly across browsers as well as features such as page navigation.  Whilst some of these differences can be impossible to resolve, by cross-browser testing we can ensure our users are able to access all content and execute all functionality without any major issues.
+## **Testing Considerations**
+
+As there are so many combinations of screen sizes, devices, operating systems and browsers to choose from, and limited time and resource to test, it is important that we establish the intended range of configurations that we support to ensure the best coverage of our users.
+Ideally, it is best to view and test these applications on physical devices with the same configuration as the end user.  However, in practice this is not always possible and therefore we use BrowserStack.  BrowserStack is a cloud-based cross-browser testing tool that enables us to test our websites across various browsers on different operating systems and mobile devices without having to install virtual machines or emulators.
+
+## **What we support**
+Evaluating the analytics available to us, it has been agreed to support the following as a minimum:
+
+### **Quartex Applications**
+### DAM & CMS
+
+&nbsp;&nbsp; Browsers:
+- Chrome / Edge 
+- Firefox  
+- Safari  
+- IE (*until end-of-life 15/06/22?*) 
+ 
+&nbsp;&nbsp; Operating Systems: 
+- Windows 
+- IOS 
```

Comment:
@KarenFarrell36 - the Operating Systems looks a bit inconsistent to me. iOS is Apple's OS for mobile/tablet, should this be MacOS (Apple's desktop/laptop OS)?